`e test` wrong module path error

When running `e test --runner=native` it throws an error.

`Error: Cannot find module '/Users/h/.electron_build_tools/src/utils/e`

Looks like the offending line is here.
https://github.com/electron/build-tools/blob/60f81c2226ff4c963d89ec2da96e5d25e1d743e1/src/utils/headers.js#L26

Should probably be `path.resolve(__dirname, '../e')`, alt. future proofing to get the caller directory with the `parent-module`.

--- a/src/utils/headers.js
+++ b/src/utils/headers.js
@@ -23,7 +23,7 @@ function ensureNodeHeaders(config) {
 
   if (needs_build) {
     const exec = process.execPath;
-    const args = [path.resolve(__dirname, 'e'), 'build', 'node:headers'];
+    const args = [path.resolve(__dirname, '..', 'e'), 'build', 'node:headers'];
     const opts = { stdio: 'inherit', encoding: 'utf8' };
     childProcess.execFileSync(exec, args, opts);
   }

